dashboard/app/views/levels/_project_ui.html.haml

```diff
@@ -0,0 +1,103 @@
+- if @level.properties['free_play'] == 'true'
```

Comment:
Unfortunately this won't work for legacy levels which pull `appOptions.level.freePlay` from the [`levels.js`](https://github.com/code-dot-org/code-dot-org/blob/8158422b4ec427807d79f3cf7ff996b01cd03109/apps/src/studio/levels.js#L1194) file.  For example `@level.properties['free_play']` isn't set for http://studio.code.org/s/playlab/stage/1/puzzle/10.